### PR TITLE
Added MODIS hero images

### PR DESCRIPTION
**Pull request by pholleway**

### CODE REVIEW

**Comment by pholleway** [comment]
- Slight color map changes will be made on 13a1/13q1 (veg index) and 15a2h/15a3h (leaf area index) to help with variety throughout images. 
- There are 2 data products missing. 14a1 and 09Q1. I’ll add them ASAP.

**Comment by mmcfarland** [comment]
Thanks @pholleway, these look great! Two general comments:

* Confusingly, we don't actually put the thumbnails in this repo. Since they're referenced by the STAC collection directly, we store them outside of this frontend deployment. I can take care of moving them to our CDN for that purpose, if you want to share a zip file in teams. The heros _are_ a frontend asset, so we do keep them here. The next comment also applies to these thumbs, regardless.
* We can typically get a good compression on PNG/JPGs produced from image editing software. I like to use tinypng.com, and as an example I took a hero image from this set from 1.5mb to 370kb which will greatly improve loading times on the website. Can you run these images through that site to slim them down?

**Comment by pholleway** [comment]
> Thanks @pholleway, these look great! Two general comments:
> 
> * Confusingly, we don't actually put the thumbnails in this repo. Since they're referenced by the STAC collection directly, we store them outside of this frontend deployment. I can take care of moving them to our CDN for that purpose, if you want to share a zip file in teams. The heros _are_ a frontend asset, so we do keep them here. The next comment also applies to these thumbs, regardless.
> * We can typically get a good compression on PNG/JPGs produced from image editing software. I like to use tinypng.com, and as an example I took a hero image from this set from 1.5mb to 370kb which will greatly improve loading times on the website. Can you run these images through that site to slim them down?

Thanks for the info, Matt! I'll send the compressed files to you on Teams by Wednesday 3/30. I'll be out of the office through Tuesday.

File: src/config/datasets.yml [hunk 1]
```diff
@@ -300,25 +300,81 @@ collections:
   landsat-8-c2-l2:
     isHidden: true
 
+  modis-09A1-061:
```
Comment (mmcfarland) [inline src/config/datasets.yml:303]: @pholleway I added this initial config that matches a hero image to a collection and adds a category (we can change these if they're not right). This is also where references to the notebooks will go when they're ready.

**Pull request merged**